sdks/csharp/.github/workflows/test.yml

```diff
@@ -0,0 +1,172 @@
+name: Test Suite
+
+on:
+  push:
+    branches:
+      - staging
+  pull_request:
+
+jobs:
+  unity-testsuite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      # Grab the SpacetimeDB branch name from the PR description. If it's not found, master will be used instead.
```

Comment:
IMO We no longer need to extract the SpacetimeDB branch for this CI step

Reply:
Agreed, see https://github.com/clockworklabs/SpacetimeDB/pull/2952